ibis-server/tests/routers/v2/connector/test_mysql.py

```diff
@@ -475,6 +475,80 @@ async def test_connection_valid_ssl_mode(client, mysql_ssl_off: MySqlContainer):
     assert response.text == '"8.0.40"'
 
 
+# MySQL specific chinese tests
+# So we dont mess up with the other tests's connection
+async def test_chinese_column_name_and_data(client):
```

Comment:
I found that this case doesn't cover the change
```
        kwargs.setdefault("charset", "utf8mb4")
```
I tried to remove the charset setting. The test is still passed.
Could you double-check it?